scheduler/src/cook/mesos/util.clj

```diff
@@ -375,16 +389,32 @@
 (timers/deftimer [cook-mesos scheduler get-running-tasks-duration])
 
 (defn get-running-task-ents
-  "Returns all running task entities"
+  "Returns all running task entities."
   [db]
   (timers/time!
     get-running-tasks-duration
-    (->> (q '[:find ?i
+    (->> (q '[:find [?i ...]
```

Comment:
Why was this change needed?

Reply:
I think this is what we meant to have originally. This lets us avoid unpacking the single-element tuples that were being returned by the original query.
http://docs.datomic.com/query.html#find-specifications